### ISSUE TITLE
this should definitely work

### ISSUE THREAD

**Issue by goldingn** [greta-dev/greta#326]
but it doesn't:

```r
library(greta)
x <- normal(0, 1, dim = c(2, 2, 2))
m <- model(x)
draws <- mcmc(m)
```

It works when the dims are all 1s, or with fewer than 3 dimensions. I guess this is either a TFP distribution broadcasting issue, or related to #298

**Comment by goldingn** [comment]
Error is from python, and looks like:
> ```
> Incompatible shapes: [4,2,2,2] vs. [4,1,1]
> ```

**Comment by goldingn** [comment]
Closed by #329
